src/Language/PureScript/Pretty/Types.hs

```diff
@@ -74,6 +74,12 @@ prettyPrintRowWith tro open close = uncurry listToBox . toList []
 prettyPrintRow :: Type -> String
 prettyPrintRow = render . prettyPrintRowWith defaultOptions '(' ')'
 
+proxyType :: Pattern () Type (Type, Type)
+proxyType = mkPattern match
+  where
+  match (ProxyType t) = Just (TypeConstructor (Qualified Nothing (ProperName "@")), t)
```

Comment:
Interesting trick :)

Reply:
I'm wondering if we should add `Prefix` to `pattern-arrows` going forward, but I'm fine with this in the meantime. Could you please add a comment though?